src/AWSS3.jl

```diff
@@ -646,7 +657,9 @@ function s3_get_tags(aws::AbstractAWSConfig, bucket, path=""; kwargs...)
     end
 end
 
-s3_get_tags(a...; b...) = s3_get_tags(global_aws_config(), a...; b...)
+function s3_get_tags(bucket, path=""; kwargs...)
+    return s3_get_tags(global_aws_config(), bucket, path; kwargs...)
+end
```

Comment:
I see you're specifying `path=""` to avoid defining two methods. Usually in this case I'd use `args...` as it avoids having to set the default value twice leading to the potential for them to become different over time. Alternatively, you could use a `const` and reference it in both places:
```suggestion
s3_get_tags(args...; kwargs...) = s3_get_tags(global_aws_config(), args...; kwargs...)
```